<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_choose_cou_detail</class>
 <widget class="QWidget" name="admin_choose_cou_detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="2,8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="1">
          <widget class="QLineEdit" name="ld_detail_sname"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_37">
           <property name="text">
            <string>学生姓名</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="ld_detail_sno"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_42">
           <property name="text">
            <string>学生学号</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_detail_delete">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_detail_query">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btn_detail_add">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btn_detail_reset">
           <property name="text">
            <string>重置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QTableWidget" name="table_detail">
           <column>
            <property name="text">
             <string>学生姓名</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>学生学号</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
